### DISCUSSION TITLE
[Prettier Plugin]

### DISCUSSION THREAD

**Discussion by bu3alwa** [tailwindlabs/tailwindcss#11336] [Help]
```
89:29  Warning: Replace `·flex-col` with `-col·flex`  prettier/prettier
90:31  Warning: Replace `·flex-col` with `-col·flex`  prettier/prettier
106:31  Warning: Replace `·flex-col` with `-col·flex`  prettier/prettier
122:31  Warning: Replace `·flex-col` with `-col·flex`  prettier/prettier
138:31  Warning: Replace `·flex-col` with `-col·flex`  prettier/prettier
```

I don't understand why this happens when I add the prettier tailwind plugin. I am running it with eslint and those warnings only show up when the tailwindcss prettier plugin is added. Can someone help with this issue?

**Comment by Robertclive54** [comment]
I read your Question i have good knowledge about it so that my opinion on it actually , The warnings you are seeing related to ·flex-col and -col·flex when using the Tailwind CSS Prettier plugin are likely due to a conflict between the plugin and your specific code configuration. It's possible that the plugin is making formatting changes that are conflicting with your existing code style or configuration. To resolve this issue, you can try the following steps: Check your ESLint configuration and rules to see if there are any conflicts or overrides related to the formatting or the use of flexbox classes in your code. [mymilestonecard  ](https://www.mymilestonecard.biz/)Review the configuration of the Tailwind CSS Prettier plugin and ensure it aligns with your desired code style and conventions. Consider adjusting the rules or settings of either ESLint or the plugin to better match your preferred code formatting.
